app/models/planet.py

```diff
@@ -0,0 +1,29 @@
+from app import db
+
+class Planet(db.Model):
+    id = db.Column(db.Integer, primary_key = True, autoincrement = True)
+    name = db.Column(db.String)
+    description = db.Column(db.String)
+    moon_count = db.Column(db.Integer)
+
+    def to_json(self):
+        return {
+            "id": self.id, 
+            "name": self.name,
+            "description": self.description, 
+            "moon_count": self.moon_count
+        }
+
+    def update(self, req_body):
+        self.name = req_body["name"]
+        self.description = req_body["description"]
+        self.moon_count = req_body["moon_count"]
```

Comment:
Same thought here about validating the PUT request fields that your server needs to handle as with POST. How can you validate and handle the KeyError if a field is not present in the request?